cloud-info/ams-wrapper.sh

```diff
@@ -100,4 +98,21 @@ printf '"}]}' >>ams-payload
 
 curl -X POST "$ARGO_URL" -H "content-type: application/json" -d @ams-payload
 
+if [ -f site.json ]; then
+	# Put this info into S3, configure rclone config with
+	# a provider named "s3" using env variables
+	export RCLONE_CONFIG_S3_TYPE=s3
+	RCLONE_CONFIG_S3_ACCESS_KEY_ID="$(yq -r '.s3.access_key_id' <"$CHECKIN_SECRETS_FILE")"
+	export RCLONE_CONFIG_S3_ACCESS_KEY_ID
+	RCLONE_CONFIG_S3_SECRET_ACCESS_KEY="$(yq -r '.s3.secret_access_key' <"$CHECKIN_SECRETS_FILE")"
+	export RCLONE_CONFIG_S3_SECRET_ACCESS_KEY
+	RCLONE_CONFIG_S3_ENDPOINT="$(yq -r '.s3.endpoint' <"$CHECKIN_SECRETS_FILE")"
+	export RCLONE_CONFIG_S3_ENDPOINT
+	S3_BUCKET_NAME="$(yq -r '.s3.bucket' <"$CHECKIN_SECRETS_FILE")"
+	export S3_BUCKET_NAME
+	export RCLONE_CONFIG_S3_ACL=private
+	export RCLONE_CONFIG_S3_NO_CHECK_BUCKET=true
+	rclone copy site.json "s3:$S3_BUCKET_NAME/$SITE_NAME"
```

Comment:
I would add a decision record for using rclone instead of Wrangler https://developers.cloudflare.com/r2/objects/upload-objects/

Reply:
I still want to be cloudflare agnostic, rclone was the tool I was aware that could be used without much hassle. 
BTW, we don't have decision records (yet). I guess if we want to get started with them, they should come as part of the PR, no?

Reply:
I'm also in favour of using well known generic and independent tools like rclone instead of cloudflare owned stuff, at least when it can be avoided and does not have a clear/decisive benefit.

Reply:
that's fine, but write a decision record for it (using _e.g._ https://github.com/npryce/adr-tools)

Reply:
That's interesting.
It would be good to document this kind of things/guidelines/suggestions for repo management somewhere.
We could start by adding it to some internal page for GitHub.

Reply:
Added a note to https://confluence.egi.eu/display/Office/GitHub

Reply:
Thanks!
Lately I realised that we could also use the repo template to document this: https://github.com/EGI-Federation/repository-template